.travis.yml

```diff
@@ -45,6 +45,7 @@ env:
     - TRAVIS_FLAVOR=consul FLAVOR_VERSION=v0.6.4
     - TRAVIS_FLAVOR=consul FLAVOR_VERSION=0.7.2
     - TRAVIS_FLAVOR=docker_daemon
+    - TRAVIS_FLAVOR=kubernetes
```

Comment:
Can you add this in the proper alphabetical order? It helps in auto resolving merge conflicts